### PR TITLE
Add experimental platform and toolchain definitions.

### PR DESCRIPTION
**Pull request by xingao267**
- experimental rbe_debian8 platform and toolchain definitions
- experimental ubuntu_gcc_fully_loaded platform and toolchain
definitions

### CODE REVIEW

File: platform/experimental/BUILD [hunk 1]
```diff
@@ -0,0 +1,49 @@
+package(default_visibility = ["//visibility:public"])
```
Comment (nlopezgi) [inline platform/experimental/BUILD]: rename directory 'platforms'

Comment (nlopezgi) [reply platform/experimental/BUILD]: set license headers

File: platform/experimental/BUILD [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        "@bazel_tools//tools/cpp:gcc",
+        ":trusty",
+    ],
+    remote_execution_properties = """
+          properties: {
+            name: "container-image"
+            value:"docker://gcr.io/asci-toolchain/nosla-ubuntu-gcc-fl@sha256:20bf8513ba378ff3288904ab73b9993586c731e42855e27cf3be42c785a42489"
+         }
+         """,
+)
```
Comment (nlopezgi) [inline platform/experimental/BUILD]: Lets only add the debian8 for now and only migrate the debian8 targets

File: platform/experimental/BUILD (hunk 1 above)
Comment (xingao267) [reply platform/experimental/BUILD]: done.
Also rename toolchain to toolchains if that's fine with you

File: platform/experimental/BUILD (hunk 2 above)
Comment (xingao267) [reply platform/experimental/BUILD]: done

**Pull request merged**